src/components/CreditCard/CreditCard.jsx

```diff
@@ -0,0 +1,16 @@
+function CreditCard(props) {
+    const {type, number, expirationMonth, expirationYear, bank, owner, bgColor, color} = props
+    
+    return (
+    <div style={{color: `${color}`, background: `${bgColor}` }}>
+    <p> {type}</p>
+    <p> {number}</p>
```

Comment:
Here you were just missing rendering multiple "***" followed by the last 4 numbers of the card